src/Uplink/Admin/License_Field.php

```diff
@@ -91,14 +91,15 @@ public function render( bool $show_title = true, bool $show_button = true ) {
 	 * @return void
 	 */
 	public function enqueue_assets() {
-		$handle = 'stellarwp-uplink-license-admin';
+		$handle = sprintf( 'stellarwp-uplink-license-admin-%s', Config::get_hook_prefix() );
 		$path   = preg_replace( '/.*\/vendor/', plugin_dir_url( $this->get_plugin()->get_path() ) . 'vendor', dirname( __DIR__, 2 ) );
 		$js_src = apply_filters( 'stellarwp/uplink/' . Config::get_hook_prefix() . '/admin_js_source', $path . '/assets/js/key-admin.js' );
-
 		wp_register_script( $handle, $js_src, [ 'jquery' ], '1.0.0', true );
 		wp_enqueue_script( $handle );
+		$action_postfix = strtolower( str_replace( '-', '_', sanitize_title( Config::get_hook_prefix() ) ) );
```

Comment:
We process the slug like this in a few places, should we instead add a config method? `Config::get_hook_prefix_underscored()` or similar?

Reply:
Done